zh/common.json

```
@@ -2047,800 +581,1383 @@
+  "You can have up to 3 backups: One is automatically created whenever Vortex starts up with no issue, one is automatically created hourly (while using Vortex) and one you can create manually.": "您最多可以有3个备份。其中一个是在Vortex正常启动时自动创建；一个是每小时自动创建一次（在使用Vortex时）；另一个您可以手动创建。",
+  "Please clone this theme to modify it.": "请克隆此主题以进行修改。",
+  "Read system fonts": "读取系统字体",
+  "System Fonts": "系统字体",
+  "Makes all system fonts installed on the system available in the Font dropdowns. This function seems to cause Vortex to crash for a very small number of users and we have not been able to identify what sets the affected systems apart yet.": "使所有系统已安装字体在“字体”下拉列表中可用，这个功能会导致极少数用户的Vortex崩溃，并且我们还无法确认受影响系统有什么不同。",
+  "The quick brown fox jumps over the lazy dog": "敏捷的棕色狐狸跨过懒狗",
+  "Dashlet Height:": "仪表板高度：",
+  "Dashlet Height": "仪表板高度",
+  "Every dashlet (the widgets on the Dashboards) has a height that is a multiple of this value and a width of either 1/3, 2/3 or 3/3 of the window width. Here you can adjust the base height of dashlets but we can't promise every dashlet will look good or even be usable with non-default height.": "每个仪表板（仪表板上的小部件）的高度是该值的倍数，宽度是窗口宽度的1/3、2/3或3/3。在这里，您可以调整仪表盘的基准高度，但我们不能保证在非默认高度下每个仪表盘都能正常、完美地显示。",
+  "Extension": "扩展",
+  "Network is offline": "网络处于离线状态",
+  "Drop {{ accept }}": "拖入{{ accept }}",
+  "Check for Updates": "检查更新",
+  "Group the table by this attribute": "按此属性对表格进行分组",
+  "Expand all": "展开全部",
+  "Collapse all": "收起全部",
+  "Endorsement disabled by author": "作者已禁用点赞",
+  "Please understand that this is only for feedback for the \"Vortex\" application, all feedback regarding the website \"Nexus Mods\" will be ignored.": "请明白这仅用于\"Vortex\"应用程序的反馈，有关网站\"Nexus Mods\"的所有反馈将被忽略。",
+  "Crash": "程序崩溃",
+  "Login Problem": "登录问题",
+  "Slow Downloads": "下载速度慢",
+  "Other": "其他",
+  "feedback-instructions": "请注意以下几点,<1></1><2><0>请使用标点符号以及换行</0><1>请使用英语</1><2>请尽可能简洁精准的描述问题。您甚至不用写成一个完整的句子。因为漏洞反馈是一个技术性文档，不是散文</2><3>一个反馈只能描述一个问题</3><4>请不要进行盲目猜测或立即下您自己的结论。只需要描述您现在看到的以及您希望看到的</4><5>下面提供了一个关于如何报告Bug的范例，请参考</5></2>修复Bug是个极其耗费时间的工作。如果您能将反馈写的尽可能好，我们就能花更少的时间在这上面，从而拥有更多的精力将Vortex做的更好！",
+  "Nexus downloads are capped at 1-2MB/s - Go Premium for uncapped download speeds": "Nexus的下载速度上限为1-2MB/s，Premium会员的下载速度则没有上限",
+  "Current Game": "当前游戏",
+  "Tools & Extensions": "工具 & 扩展",
+  "Managed ({{filterCount}})": "管理： ({{filterCount}})",
```

```suggestion
  "Managed ({{filterCount}})": "已管理： ({{filterCount}})",
```
我不太确定中文标点后面的空格是否需要或者可以删去。